drivers/ethernet/eth_enc28j60.c

```diff
@@ -850,6 +860,7 @@ static int eth_enc28j60_init(const struct device *dev)
 		.full_duplex = DT_INST_PROP(0, full_duplex),                                       \
 		.timeout = CONFIG_ETH_ENC28J60_TIMEOUT,                                            \
 		.hw_rx_filter = DT_INST_PROP_OR(inst, hw_rx_filter, ENC28J60_RECEIVE_FILTERS),     \
+		.random_mac = DT_INST_PROP(inst, zephyr_random_mac_address),                    \
```

Comment:
I do assume that the 'zephyr,random-mac-address' is well established, Zephyr wide property? Or is it only one-off?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/b82b5b0734c30490368e21627423f10036487343/dts/bindings/ethernet/ethernet-controller.yaml#L13